api/src/main/java/org/openmrs/Condition.java

```diff
@@ -396,4 +396,10 @@ public boolean equals(Object o) {
 		}
 		return Objects.equals(endReason, conditionToBeChecked.endReason);
 	}
+	
+	@Override
+	protected void markAsDirty(Object oldValue, Object newValue) {
+		// Do nothing
+	}
```

Comment:
In light of the above, this will not be necessary either.

Reply:
Removed.